### ISSUE TITLE
unable to dl youtube video 

### ISSUE THREAD

**Issue by zartlike** [ytdl-org/youtube-dl#28739]
![youtube-dl](https://user-images.githubusercontent.com/79120359/114327267-b4b55f00-9b7b-11eb-8b57-fcc3a12c663e.png)

Just installed from Snap 12-04-2021
Ubuntu 20.04 OS

Any help? Thanks

**Comment by FrankHu-RDS** [comment]
Same, I'm also having this issue.

**Comment by zartlike** [comment]
Hi dstftw,
You closed my request with imcomplete label. What does it mean?
All downloading were incomplete, right. What's the cause of it?

**Comment by kikuyan** [comment]
You should confirm that you are using the latest version of youtube-dl. It's good to read the FAQ and BUGS sections of the [README](https://github.com/ytdl-org/youtube-dl/blob/master/README.md#readme).

**Comment by zartlike** [comment]
Thanks for the advice, kikuyan.
I also created config file and pasted those example commands on that READ ME page and saved it. After that I tried to download it, it says connection refused.
Latest version checked attached
![youtube_dl](https://user-images.githubusercontent.com/79120359/114626752-7e9cea00-9cf7-11eb-8b13-f6144f35de53.png)
![youtube_dl1](https://user-images.githubusercontent.com/79120359/114626774-865c8e80-9cf7-11eb-8fab-e5901dd0aa81.png)
